lib/pharos/up_command.rb

```diff
@@ -10,7 +10,7 @@ class UpCommand < Pharos::Command
       end
     end
 
-    option '--hosts-from-tf', 'PATH', 'Path to terraform output json' do |config_path|
+    option '--tf-json', 'PATH', 'Path to terraform output json' do |config_path|
```

Comment:
Note: breaking change

Reply:
👍

Reply:
Yes, this makes sense because it's no longer just `pharos_hosts`, but also `pharos_api`.